src/main/java/de/adesso/anki/battle/GameEngine.java

```diff
@@ -49,6 +49,8 @@ public void start() {
 
     @Scheduled(fixedRate = 50)
     public void gameLoop() {
+        System.out.println("Sonar should annotate this");

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)